kratos/includes/constitutive_law.h

```diff
@@ -58,9 +58,10 @@ class KRATOS_API(KRATOS_CORE) ConstitutiveLaw : public Flags
         StrainMeasure_Hencky_Spatial,  //strain measure current   configuration
 
         // Deformation measures:
-        StrainMeasure_Deformation_Gradient, //deformation gradient as a strain measure
+        StrainMeasure_Deformation_Gradient, //material deformation gradient as a strain measure
         StrainMeasure_Right_CauchyGreen,    //right cauchy-green tensor as a strain measure
-        StrainMeasure_Left_CauchyGreen      //left  cauchy-green tensor as a strain measure
+        StrainMeasure_Left_CauchyGreen,     //left  cauchy-green tensor as a strain measure
+        StrainMeasure_Velocity_Gradient     //spatial velocity gradient as a strain measure
```

Comment:
I don't remember having reached an agreement about this.

Reply:
Well, the list is incomplete, you can chose the name you prefer.

Reply:
let's have this to go in as a separate PR so that if someone has comments they can be made

Reply:
i think i can agree on this. Any problem from @KratosMultiphysics/structural-mechanics ?